stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.36.0
+version: 1.36.1
```

Comment:
Need a bump as another PR got in first.

Reply:
@steven-sheehy yep, I noticed that and wasn't sure if either or both would be accepted, then the other would conflict and I'll resolve right after that.